content/go/server.go.md

```diff
@@ -0,0 +1,522 @@
+---
+title: "More Bare Metal Server examples"
+description: "Working with Bare Metal Server"
+date: "2021-07-12"
+classes: 
+    - "SoftLayer_Container_Product_Order"
+    - "SoftLayer_Product_Item_Price"
+    - "SoftLayer_Hardware_Server"
+    - "SoftLayer_Product_Package"
+    - "SoftLayer_Container_Product_Order_Hardware_Server"
+    - "SoftLayer_Product_Order"
+tags:
+    - "baremetalservers"
+    - "hardware"
+    - "order"
+    - "objectfilter"
+    - "objectmask"
+---
+
+A few examples on interacting with Bare Metal Server, [SoftLayer_Hardware_Server](https://sldn.softlayer.com/reference/services/SoftLayer_Hardware_Server/).
+
+```go
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/IBM-Cloud/ibm-cloud-cli-sdk/bluemix/terminal"
+	"github.com/softlayer/softlayer-go/datatypes"
+	"github.com/softlayer/softlayer-go/filter"
+	"github.com/softlayer/softlayer-go/services"
+	"github.com/softlayer/softlayer-go/session"
+	"github.com/softlayer/softlayer-go/sl"
+)
+
+// Session created using values set in the environment, or from the local configuration file (i.e. ~/.softlayer).
+var sess = session.New()
+
+type Dictionary map[string]string
+
+func main() {
+
+	//Shows all servers in the account.
+	listServers()
+
+	//Shows all available packages to order/create a Bare Metal server.
+	listServerPackages()
+
+	//Shows just billing hourly packages to order/create a Bare Metal server.
+	listServerPackagesHourly()
+
+	//Shows the items within the package
+	packageKeyname := "DUAL_E52600_V4_12_DRIVES"
+	listItemsByPackageKeyname(packageKeyname)
```

Comment:
I see that you are printing the items detail, but you are not showing the prices, I think this is important to the customer to know the cost for each item price, hourly or monthly for the ordering. I suggest you to add a method to print this.
Take in count that you have to be careful with the capacityRestriction too.

Reply:
added capacityRestriction and prices.